jsonb-generator/src/main/java/io/avaje/jsonb/generator/ValuePrism.java

```diff
@@ -15,6 +15,6 @@
     name = "JacksonValuePrism",
     superInterfaces = ValuePrism.class)
 public interface ValuePrism {
```

Comment:
So in this case, `ValuePrism` is just being used
- to maintain consistent organization
- to house the constants instead of using `Constants`

Would it be preferred to use `PRISM_TYPE` for some of those too?
e.g.
`static final String JSON_IMPORT = "io.avaje.jsonb.Json.Import";`
could be
`static final String JSON_IMPORT = ImportPrism.PRISM_TYPE;`

Reply:
(pushed that commit for your review)